### DISCUSSION TITLE
Make it possible to specify pages in prerender

### DISCUSSION THREAD

**Discussion by zim32** [vikejs/vike#597] [Discussion] [answered]
My case is that we have very big site and we want to prerender only some of them dynamically (f.e. user select some pages and click prerender).

As I understand  using programmatic prerender it is not possible to specify pages.

Maybe there is some workaround?

**Comment by samuelstroschein** [comment, marked as answer]
You might be looking for the `doNotPrerender` option https://vite-plugin-ssr.com/doNotPrerender
